COMMUNITY-ROLES.md

```diff
@@ -211,7 +203,7 @@ and are responsible for performing role changes in the community.
 ### Past owners
 
 The following people used to be owners in the tldr-pages organization,
-and have since moved on to other projects.
```

Comment:
typo at "otherwise" :smile:

Reply:
Haha thanks, let me fix it up real quick :)